x-pack/plugins/apm/dev_docs/testing.md

```diff
@@ -23,17 +23,44 @@ API tests are separated in two suites:
 - a basic license test suite [default]
 - a trial license test suite (the equivalent of gold+)
 
+### Run tests with [--trial] license
+
 ```
 node scripts/test/api [--trial] [--help]
 ```
 
+The above command will initiate an Elasticsearch instance on http://localhost:9220 and a kibana instance on http://localhost:5620 and will run the api test against these environments.
+Once the tests finish, the instances will be terminated.
+
+### Start test server
```

Comment:
"Start test  server" / "Run all tests" section could be moved under a debugging section or something similar so the difference to `node scripts/test/api` is more clear

Reply:
From my understanding, the difference is that once the tests finish the instances(es & kibana) will be terminated in cases you run `node scripts/test/api` script and I mention that on _Run tests with [--trial] license_

Reply:
Nice one 👍